examples/kubernetes/connectivity-check/main_tool.cue

```diff
@@ -75,8 +76,20 @@ ccCommand: {
 		}
 	}
 
+	task: filterTraffic: {
+        if #flags.traffic == "internal" {
+            resources: [ for x in task.filterName.resources if x.metadata.labels.traffic != "external"  {x}]
+        }
+		if #flags.traffic == "external" {
+            resources: [ for x in task.filterName.resources if x.metadata.labels.traffic != "internal"  {x}]
+        }
+		if #flags.traffic == "any" {
```

Comment:
nit: There looks like there's something funny going on here with the spacing.

One of my TODO's is to add in `cue fmt` to help format cue files in a consistent way, I suspect that'll modify the file here.

Reply:
oh nice, I think it's due to my Goland setup with tab and space. Will run cue fmt, thanks